### PR TITLE
[pfto] Add softmax test

### PR DESCRIPTION
**Pull request by take-cheeze**

### CODE REVIEW

**Comment by take-cheeze** [comment]
Failure seems to be caused by https://github.com/pfnet/pytorch-pfn-extras/pull/639

**Comment by emcastillo** [comment]
CI unrelated, lets merge
